freemocap/core_processes/export_data/blender_stuff/get_best_guess_of_blender_path.py

```diff
@@ -8,7 +8,11 @@
 
 def guess_blender_exe_path_from_path(base_path: Union[str, Path]) -> Optional[Path]:
     base_path = Path(base_path)
-    blender_folder_paths = [path for path in base_path.rglob("blender.exe")]
+    try:
+        blender_folder_paths = [path for path in base_path.rglob("blender.exe")]
+    except OSError:
+        logger.info(f"Unable to access: {str(base_path)}")
```

Comment:
```suggestion
        logger.error(f"Unable to access: {str(base_path)}")
```

This should be logged as an error, right? or at least a warning?

Reply:
I'm trying to think of circumstances where this kind of error would arise - do you know any specifics?